packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: External Identity Providers
+---
+# External Identity Providers
+As a developer building a custom application, you want to give your users the freedom to choose which Identity Provider that they use to sign in to your application. But first you should understand how various Identity Providers connect to Okta. 
+
+What is an Identity Provider? It is a service that creates and maintains identity information and then provides authentication services to your applications. Identity Providers can significantly reduce sign-in and registration friction, which allows your users to access applications with one click and without needing to create new passwords or remember usernames. 
+
+## The big picture
+Okta manages connections to other Identity Providers for your application and sits between your application and the Identity Provider that authenticates your users. 
+
+* When you use Okta as the [user store](https://developer.okta.com/product/user-management/) for your applications, users can sign in with their email and password by default. See our guides for how to sign in users to your [web](/docs/guides/sign-into-web-app/), [mobile](/docs/guides/sign-into-mobile-app/), and [single-page](/docs/guides/sign-into-spa/) apps.
+
+* You can add connections to Identity Providers like Twitter or Facebook. This is called social login or social authentication. It allows your users to sign in to your app using credentials from their existing accounts with those Identity Providers. After users authenticate, you sync their accounts into your Okta Universal Directory while continuing to use that Identity Provider for user authentication. This eliminates the need to store an additional username and password for that user. 
```

Comment:
I know exactly what you mean here, but I'm thinking it might be simpler to remove the use if the word "accounts"? Since its slightly synonomous with "credentials" already. 

Could we mean the same thing by just dropping it and saying:

"It allows your users to sign in to your app using credentials from their existing Identity Providers. After users authenticate, you sync their existing IDP credentials into your Okta Universal Directory by continuing to use that Identity Provider for user authentication.

Reply:
@albertchen-okta 

That's fine, one question about a change in the second sentence:

After users authenticate, you sync their existing Identity Provider credentials into your Okta Universal Directory WHILE continuing to use that Identity Provider for user authentication.

Please correct me if I'm wrong, but the first time they sign in using their existing Identity Provider credentials, their credentials get synced into the Okta Universal Directory, right?   I think saying "BY continuing to use that Identity Provider" implies that they sync their credentials with Okta ONLY by continuing to use the Identity Provider for user authentication....but maybe I don't understand it like I thought I did.

Reply:
Also - are you suggesting dropping the last sentence? Thanks!

Reply:
* "while" works
* I like the last sentence, I just didn't type it sorry. "This eliminates the need to store an additional username and password for that user. "